osgar/bus.py

```diff
@@ -102,8 +120,8 @@ def listen(self):
         packet = self.queue.get()
         if packet is None:
             raise BusShutdownException()
-        timestamp, channel, data = packet
-        return timestamp, channel, data
+        self.time, channel, data = packet
```

Comment:
is it good idea? `self.time` is then set on several places? (I remember at least `class Node`)

Reply:
(but this is some other `self.time` so probably ignore)